docs/docs/migration-guide.mdx

```diff
@@ -10,6 +10,17 @@ description: |
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+## Rasa 2.5 to 2.6
+
+### Forms
+
+#### New `ignored_intents` parameter in Forms
+
+We added a new parameter under Forms with the name `ignored_intents`. This parameter
+can be used in case the user never wants to fill any slots of a form with the specified
```

Comment:
can we please show them an example which illustrates what they have to change exactly (like a "before" and "after" version of the domain)?